addons/mail/static/src/core/common/message_reaction_list.js

```diff
@@ -0,0 +1,88 @@
+import { useHover } from "@mail/utils/common/hooks";
+import { Component, useState } from "@odoo/owl";
+import { Dropdown } from "@web/core/dropdown/dropdown";
+import { useDropdownState } from "@web/core/dropdown/dropdown_hooks";
+
+import { _t } from "@web/core/l10n/translation";
+import { useService } from "@web/core/utils/hooks";
+
+export class MessageReactionList extends Component {
+    static template = "mail.messageReactionList";
+    static components = { Dropdown };
+    static props = ["message", "openReactionMenu", "reaction"];
+
+    setup() {
+        super.setup();
+        this.store = useState(useService("mail.store"));
+        this.ui = useService("ui");
+        this.hover = useHover(["reactionButton", "reactionList*"], () => {
+            this.preview.isOpen = this.hover.isHover;
+            if (this.hover.isHover) {
+                clearTimeout(this.showReactionListTimeout);
+                this.showReactionListTimeout = setTimeout(() => {
+                    this.state.showReactionList = true;
+                }, 25);
+            } else if (!this.hover.isHover) {
+                clearTimeout(this.showReactionListTimeout);
+                this.state.showReactionList = false;
+            }
+        });
+        this.preview = useDropdownState();
+        this.state = useState({ showReactionList: false, hightlight: false });
```

Comment:
`highlight` is useless, this could be fully done in CSS.
I removed it because the value is pretty low, and this avoid styling text partially which is challenging for good lang support.